crates/starknet-server/tests/common/devnet.rs

```diff
@@ -92,6 +91,7 @@ impl BackgroundDevnet {
                 .spawn()
                 .expect("Could not start background devnet");
 
+        let pid = process.id();
```

Comment:
where is this used?

Reply:
Leftover from debugging. PID is used to know to which process I shall attach debugger